testing/examples/frame_system_tests.rs

```diff
@@ -48,6 +48,10 @@ async fn main() {
 	// GetAccountInformation
 	let _account_info = api.get_account_info(&alice).unwrap().unwrap();
 	let _account_data = api.get_account_data(&alice).unwrap().unwrap();
+	// Empty account information
+	let ferdie = AccountKeyring::Two.to_account_id();
+	assert!(api.get_account_info(&ferdie).unwrap().is_none());
+	assert!(api.get_account_data(&ferdie).unwrap().is_none());
```

Comment:
```suggestion
```

Reply:
Don't we want to test it?

Reply:
Actually, you're right. I was thinking that account_data call account info anyway, but there's a `map`.. and we don't want that to change anything.